webapp/tests/Unit/Controller/Jury/JuryControllerTest.php

```diff
@@ -158,11 +162,25 @@ public function testCheckAddEntityJury(): void
         }
     }
 
+    public function helperCheckExistance(string $id, $value, array $element): void {
+        if (in_array($id, static::$addEntitiesShown)) {
+            $tmpValue = $element[$id];
+            if (is_bool($value)) {
+                $tmpValue = $value ? 'yes' : 'no';
+            }
+            self::assertSelectorExists('body:contains("' . $tmpValue . '")');
+        }
+        if (in_array($id, static::$addEntitiesCount)) {
+            self::assertSelectorExists('body:contains("' . count($element[$id]) . '")');
+        }
+    }
+
     /**
      * Test that admin can add a new entity for this controller
      */
     public function testCheckAddEntityAdmin(): void
     {
+        var_dump($this);
```

Comment:
Left over dump

Reply:
Yes I had to remove that commit, should now be reviewable.